clang/include/clang/CodeGen/CodeGenABITypes.h

```diff
@@ -75,6 +75,12 @@ const CGFunctionInfo &arrangeCXXMethodType(CodeGenModule &CGM,
                                            const FunctionProtoType *FTP,
                                            const CXXMethodDecl *MD);
 
+const CGFunctionInfo &arrangeCXXMethodCall(CodeGenModule &CGM,
+                                           CanQualType returnType,
+                                           ArrayRef<CanQualType> argTypes,
+                                           FunctionType::ExtInfo info,
+                                           RequiredArgs args);
```

Comment:
Hmm.  The internal function for this now takes an array of `ExtParameterInfo`s.  As long as we're introducing new API, could you go ahead and add that parameter to these functions?  You'll need to either add overloads or give it a default argument (an empty array is an acceptable default).

Reply:
@rjmccall Updated. Do you mean like the latest revision?